packages/fx-core/src/component/resource/botService/appStudio/appStudioClient.ts

```diff
@@ -89,32 +90,36 @@ export class AppStudioClient {
     context?: ResourceContextV3
   ): Promise<void> {
     const axiosInstance = AppStudioClient.newAxiosInstance(token);
-    let create = false;
-    let response = undefined;
-    try {
-      if (registration.botId) {
-        const botReg = await AppStudioClient.getBotRegistration(token, registration.botId);
-        if (botReg) {
-          context?.logProvider?.info(Messages.BotResourceExist("Appstudio"));
-          return;
-        }
+
+    if (registration.botId) {
+      const botReg = await AppStudioClient.getBotRegistration(token, registration.botId);
+      if (botReg) {
+        context?.logProvider?.info(Messages.BotResourceExist("Appstudio"));
+        return;
+      } else {
+        // Potential case when developers switched their M365 accounts.
```

Comment:
If it is possibly caused by switching account, should this info be included in error message to give user a hint?

Reply:
This error will be catched on upper layer and be transformed to another error which will include the hint.
I've discussed with Yui which is the swtichAccountError owner, and I may update some part of her code later . Will ping you for review when it's ready.